cogs/error.py

```diff
@@ -13,6 +13,8 @@ async def on_command_error(self, ctx: commands.Context, error):
         print(error)
         if isinstance(error, NotRecruiter):
             await ctx.reply("You need the recruiter role to perform this command!")
+        elif isinstance(error, NotManager):
```

Comment:
By using the commands.has_role() we don't need this any more.

In fact, we also won't need 'NotRecruiter' error either, as the @commands.has_role() will take care of that for us.